x/pairing/client/cli/tx_modify_provider.go

```diff
@@ -101,7 +101,7 @@ func CmdModifyProvider() *cobra.Command {
 			}
 			var providerEntry *epochstoragetypes.StakeEntry
 			for idx, provider := range response.StakeEntry {
-				if provider.Address == address.String() {
+				if provider.Operator == address.String() {
```

Comment:
we need to let vault to also use this command

Reply:
done. see https://github.com/lavanet/lava/commit/8fa736e4fc672043c1744feec22aa8bc6ebec4ed